src/core/QCheck2.mli

```diff
@@ -289,6 +289,47 @@ module Gen : sig
       Shrinks towards ['0'].
   *)
 
+  val bytes_size : ?gen:char t -> int t -> bytes t
+  (** Builds a bytes generator from a (non-negative) size generator.
+      Accepts an optional character generator (the default is {!char}).
+
+      Shrinks on the number of characters first, then on the characters.
+
+      @since NEXT_RELEASE *)
+
+  val bytes : bytes t
+  (** Builds a bytes generator. Bytes size is generated by {!nat}.
+      The default character generator is {!char}.
+      See also {!bytes_of} and {!bytes_printable} for versions with
+      custom char generator.
+
+      Shrinks on the number of characters first, then on the characters.
+
+      @since NEXT_RELEASE *)
+
+  val bytes_of : char t -> bytes t
+  (** Builds a bytes generator using the given character generator.
+
+      Shrinks on the number of characters first, then on the characters.
+
+      @since NEXT_RELEASE *)
+
+  val bytes_printable : bytes t
+  (** Builds a bytes generator using the {!printable} character generator.
+
+      Shrinks on the number of characters first, then on the characters.
+
+      @since NEXT_RELEASE *)
+
+  val small_bytes : ?gen:char t -> bytes t
```

Comment:
Note that `?gen` can not be omitted with the abstraction of generators in QCheck2

Reply:
But `string_size` line 333 and `small_string` line 363 ha an optional `?gen` argument.
Do I remove them too?

Reply:
`string_size` line 333 and `small_string` line 363 also have an optional `?gen` argument.
Do I remove them too?

Reply:
Removing the tilde `~` should make the argument labeled in the signature:
```ocaml
  val small_bytes : gen:char t -> bytes t
```
(the same should probably be done for `small_string`)